content/en/docs/concepts/overview/components.md

```diff
@@ -14,140 +13,73 @@ card:
-
-{{< glossary_definition term_id="cloud-controller-manager" length="short" >}}
-
-The cloud-controller-manager only runs controllers that are specific to your cloud provider.
-If you are running Kubernetes on your own premises, or in a learning environment inside your
-own PC, the cluster does not have a cloud controller manager.
-
-As with the kube-controller-manager, the cloud-controller-manager combines several logically
-independent control loops into a single binary that you run as a single process. You can
-scale horizontally (run more than one copy) to improve performance or to help tolerate failures.
-
-The following controllers can have cloud provider dependencies:
-
-  * Node controller: For checking the cloud provider to determine if a node has been deleted in the cloud after it stops responding
-  * Route controller: For setting up routes in the underlying cloud infrastructure
-  * Service controller: For creating, updating and deleting cloud provider load balancers
-
-## Node Components
-
-Node components run on every node, maintaining running pods and providing the Kubernetes runtime environment.
-
-### kubelet
-
-{{< glossary_definition term_id="kubelet" length="all" >}}
-
-### kube-proxy
-
-{{< glossary_definition term_id="kube-proxy" length="all" >}}
-
-### Container runtime
-
-{{< glossary_definition term_id="container-runtime" length="all" >}}
+  <dt>Container runtime</dt>
+  <dd>Software responsible for running containers</dd>
+</dl>
 
 ## Addons
 
-Addons use Kubernetes resources ({{< glossary_tooltip term_id="daemonset" >}},
-{{< glossary_tooltip term_id="deployment" >}}, etc)
-to implement cluster features. Because these are providing cluster-level features, namespaced resources
-for addons belong within the `kube-system` namespace.
-
-Selected addons are described below; for an extended list of available addons, please
-see [Addons](/docs/concepts/cluster-administration/addons/).
-
-### DNS
-
-While the other addons are not strictly required, all Kubernetes clusters should have
-[cluster DNS](/docs/concepts/services-networking/dns-pod-service/), as many examples rely on it.
-
-Cluster DNS is a DNS server, in addition to the other DNS server(s) in your environment,
-which serves DNS records for Kubernetes services.
-
-Containers started by Kubernetes automatically include this DNS server in their DNS searches.
-
-### Web UI (Dashboard)
-
-[Dashboard](/docs/tasks/access-application-cluster/web-ui-dashboard/) is a general purpose,
-web-based UI for Kubernetes clusters. It allows users to manage and troubleshoot applications
-running in the cluster, as well as the cluster itself.
-
-### Container Resource Monitoring
-
-[Container Resource Monitoring](/docs/tasks/debug/debug-cluster/resource-usage-monitoring/)
-records generic time-series metrics
-about containers in a central database, and provides a UI for browsing that data.
+Addons extend the functionality of Kubernetes. A few important examples include:
```

Comment:
```suggestion
Add-ons extend the functionality of Kubernetes. A few important examples include:
```

Reply:
This suggestion is not part of the change that the PR description is summarizing, so @robert-cronin you are welcome to skip it.

Reply:
@network-charles I'll skip this change to maintain consistency with current docs as discussed above. Thanks for flagging it.